src/components/ConnectContainerDialog.tsx

```diff
@@ -23,6 +23,7 @@ export interface ConnectContainerDialogProps {
   onNextClick?: () => void;
   children: ReactNode;
   onBackClick?: () => void;
+  additionalActions?: ReactNode;
```

Comment:
Unclear if this should be additionalActions or just closeIsPrimaryAction. Both would assume that back and next aren't present.